src/components/appointments/Product.js

```diff
@@ -6,7 +6,7 @@ import {NumberField} from 'components/forms';
 import ProductSelect from './ProductSelect';
 
 const Product = ({namePrefix, index}) => (
-  <>
+  <div>
```

Comment:
needed because the CSS rules target some `:first-child` so wrapping in a div makes the vertical spacing more digestible